buildSrc/src/main/groovy/airbyte-connector-acceptance-test.gradle

```diff
@@ -1,10 +1,35 @@
 import org.gradle.api.Plugin
 import org.gradle.api.Project
 
+
 class AirbyteConnectorAcceptanceTestPlugin implements Plugin<Project> {
+    
+    // Change this value to 'dev' if you want to use a locally built connector acceptance test image
+    String CONNECTOR_ACCEPTANCE_TEST_IMAGE = 'latest'
+    
     void apply(Project project) {
+        
         project.task('connectorAcceptanceTest') {
+            dependsOn project.build
+            dependsOn project.airbyteDocker
+            
+            if (!project.hasProperty('connectorAcceptanceTestVersion')) {
+                project.ext.connectorAcceptanceTestVersion = 'latest'
```

Comment:
I'm missing some gradle knowledge here - sometimes I see this being referred as `project.ext.connectorAcceptanceTestVersion` and others as `project.connectorAcceptanceTestVersion`. What's the difference? Can they be used interchangeably?

I also see `CONNECTOR_ACCEPTANCE_TEST_IMAGE` being set to `latest` at the top and then checked on line 25 - should these usages be changed to refer to `project.connectorAcceptanceTestVersion` (or the `.ext` variant?)